packages/material/src/mui-controls/MuiSelect.tsx

```diff
@@ -43,6 +49,17 @@ export const MuiSelect = React.memo((props: EnumCellProps & WithClassname) => {
     config
   } = props;
   const appliedUiSchemaOptions = merge({}, config, uischema.options);
+  const delimiter: string = appliedUiSchemaOptions.keyValueDelimiter;
+  const keyValueLabels: Option[] = (delimiter === undefined) ?
+  options.map(option => {
+    return { key: option, value: option, label: option };
+  }) :
+  options.map(option => {
+    return option.split(delimiter, 2).map((parts: void[]) => {
+      return { key: parts[0], value: option, label: parts[1] };
```

Comment:
I usually avoid function bodies which immediately start with a `return` as this just adds visual clutter.